pkg/restore/restore_test.go

```diff
@@ -1378,7 +1408,7 @@ func TestRestoreActionAdditionalItems(t *testing.T) {
 			},
 		},
 		{
-			name:    "when using a restore namespace filter, additional items that are cluster-scoped are restored",
+			name:    "when using a restore namespace filter, additional items that are cluster-scoped are restored when IncludeClusterResources=nil",
```

Comment:
Can you please re-phrase this as a `should ...` or `should not...`?

Reply:
This is an interesting issue, how to word the tests. We don't use `should` much in the velero tests, but we never talked about why. I remember reading before about whether or not to use it. I did a quick Google search and this sums up discussions I've seen before: https://www.quora.com/Can-we-use-the-word-should-when-writing-a-test-cases-summary-in-manual-testing

Reply:
yeah, so we're not using "should" much in this test file at least, so changing this one case's wording would make it inconsistent with the rest of the cases. My approach in this kind of scenario is usually to keep consistent with the existing pattern, and revisit the broader issue separately - @ashish-amarnath does that work for you?

Reply:
I didn't see the whole file.. but then this one sneaked in I guess! https://github.com/vmware-tanzu/velero/pull/2118/files#diff-3a5824b5e6665e0247bae6406e0ad1fbR318